### PR TITLE
[13.0][ADD] portal_hide_apps

### PR DESCRIPTION
**Pull request by ChristianSantamaria**
Hides the default sale, purchase and account portal menus.

### CODE REVIEW

**Review by dalonsod — CHANGES_REQUESTED** [review]
For Purchase case, looking at

https://github.com/odoo/odoo/blob/13.0/addons/purchase/controllers/portal.py#L19

it's possible to simply reset `*_count` values via method inheritance, so view modification should be no longer needed. Could you check?

**Comment by ChristianSantamaria** [comment]
> For Purchase case, looking at
> 
> https://github.com/odoo/odoo/blob/13.0/addons/purchase/controllers/portal.py#L19
> 
> it's possible to simply reset `*_count` values via method inheritance, so view modification should be no longer needed. Could you check?

Yes, you are correct, I added to greater than the purchase order, the sale order and account invoice

File: portal_hide_apps/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    'version': '13.0.1.0.0',
+    'category': 'Website',
+    'website': 'https://github.com/solvosci/slv-general',
+    'depends': [
+        'purchase',
+        'sale',
+    ],
+    'data': [
+
+    ],
```
Comment (dalonsod) [inline portal_hide_apps/__manifest__.py]: Remove empty section

**Review by dalonsod — APPROVED** [review]
👍 Please attend cosmetic change and squash commits

**Pull request merged**